scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -87,6 +112,15 @@ Param (
     [ValidateNotNullOrEmpty()]
     [string] $EventHubConnectionString = $(Throw "Event hub connection string is required"),
 
+    [ValidateNotNullOrEmpty()]
```

Comment:
Since it allows to be null, remove this.

Reply:
This means that if a user explicitly set a null in the input or empty string it should fail. I think this is required makes it very user friendly.